### PR TITLE
Add handleStyle and minimumTrackStyle props to Range

### PR DESCRIPTION
**Pull request by gragland**
Previously the `handleStyle` and `minimumTrackStyle` props only worked with Slider. Added them to Range as well.

Maybe they could be renamed to `innerTrackStyle` and `outerTrackStyle`. It makes sense for Range, and for the slider I don't think it's too much of a stretch to consider the track to the left of the handle to be "inner".

### CODE REVIEW

**Comment by paranoidjk** [comment]
what about this,  if a Array is been provided, then we use it as style of track/handle follow the array index order, otherwise if A single style object is been provided, use it for all track/handle

```js
rangeHandleStyle: Array[Style Object] | Style Object
rangeTrackStyle: Array[Style Object] | Style Object
```

**Comment by gragland** [comment]
@paranoidjk That sounds great to me!

**Comment by paranoidjk** [comment]
@gragland would you like PR for this ? sorry that I recently have no time to do this feature

**Comment by gragland** [comment]
@paranoidjk Sure I can do a PR for this within the next couple days.

File: src/Range.jsx [hunk 1]
```diff
@@ -282,6 +282,8 @@ class Range extends React.Component {
       vertical,
       included,
       disabled,
+      minimumTrackStyle,
```
Comment (benjycui) [inline src/Range.jsx]: Just `trackStyle`

And you should update README.md at the same time.

**Comment by benjycui** [comment]
@paranoidjk just use a semantic name that you prefer.

**Comment by senkevichdv** [comment]
Any news with this feature?

**Comment by mergebandit** [comment]
Would love to see this merged!

**Comment by paranoidjk** [comment]
@gragland please add api doc on readme and demo on examples at the same time.

**Comment by benjycui** [comment]
Oops conflict...

**Comment by paranoidjk** [comment]
@billycai  You can leave your comments at here, let us konw what do you think about the api.

https://github.com/react-component/slider/issues/277

**Comment by donaldtf** [comment]
Any idea when this might be merged in?

**Comment by chongjunhong** [comment]
Hi guys, has the feature been pushed to production? Thanks :)

**Comment by paranoidjk** [comment]
Thanks for @gragland ,  I will handle this, the api will looks like this:

## `rc-slider@8.x`, which is used for `antd@2.x`, `antd-mobile@1.x`

| Name         | Type    | Default | Description |
| ------------ | ------- | ------- | ----------- |
| minimumTrackStyle | Object |`{}` | The style used for the track to the left of the button. (`only used for slider, just for compatibility , will be deprecate `) |
| maximumTrackStyle | Object | `{}` | The style used for the track to the right of the button. (`only used for slider, just for compatibility , will be deprecate `) |
| handleStyle | Array[Object] \| Object | `[{}]` | The style used for handle. (`both for slider and range, the array will be used for mutli handle follow element order`) |
| trackStyle | Array[Object] \| Object | `[{}]` | The style used for track.  . (`both for slider and range, the array will be used for mutli track follow element order`)|

## `rc-slider@9.x`, which is used for `antd@3.x`, `antd-mobile@2.x`

| Name         | Type    | Default | Description |
| ------------ | ------- | ------- | ----------- |
| handleStyle | Array[Object] \| Object | `[{}]` | The style used for handle. (`both for slider and range, the array will be used for mutli handle`) |
| trackStyle | Array[Object] \| Object | `[{}]` | The style used for track.  . (`both for slider and range, the array will be used for mutli track`)|

cc @benjycui

**Comment by paranoidjk** [comment]
I opened a PR,  look forward to you guys review https://github.com/react-component/slider/pull/281

**Pull request closed without merging**